### ISSUE TITLE
Improve GUI

### ISSUE THREAD

**Issue by SammysHP** [shagr4th/droid48#15]
The app is great, thank you. But - sorry for my style - the GUI sucks.

- the button style is flat and filled with an unrealistic gradient
- various labels from the background overlap buttons
- some labels looks different than expected (e.g. shift buttons)

I can create better SVG buttons if you want. I'd really like to help making this app even better!

**Comment by antonioleal** [comment]
I agree 100% with the above comment. Eyecandy is mandatory to improve the user base!!!

**Comment by SammysHP** [comment]
Recently I created a skin for a 50g emulator. It is mostly SVG with some filters.

![hp49gp2up](https://user-images.githubusercontent.com/922874/27450729-e923cdb2-578c-11e7-9745-2d09d46145e9.png)

If you are interested I can create one for droid48, but I'm not able to apply it because I'm not familiar with the source and how graphics are handled in it.

**Comment by antonioleal** [comment]
It's beautiful. congrats.

1 vote+ for a new droid48 gui.
